Header header
int32 self_drone_id
int32 remote_drone_id
geometry_msgs/Point dpos
geometry_msgs/Point dpos_cov

float64 dyaw
float64 dyaw_cov
bool is_2d_detect
bool is_yaw_valid
